### PR TITLE
Add gas flow computation

### PR DESCRIPTION
**Pull request by michelmegens**
Implement gas flow computation in the DSMR parser service. Gas flow computation is realized by caching the last result from a gas meter, such that the next measurement can be used to compute the increase over time between the two measurements. Linear usage is assumed.

### CODE REVIEW

File: SensateIoT.SmartEnergy.Dsmr.Parser.Common/Models/Telegram.cs [hunk 1]
```diff
@@ -42,6 +42,7 @@ public class Telegram
         public decimal GasConsumption { get; set; } = 0M;
         [Obis("0-1:24.2.1", 0), TypeConverter(typeof(ObisTimestampConverter))]
         public DateTime GasTimestamp { get; set; }
+		public bool HasGasConsumption { get; set; }
```
Comment (sensateiot-cicd) [inline SensateIoT.SmartEnergy.Dsmr.Parser.Common/Models/Telegram.cs]: Fix indentation

**Review by sensateiot-cicd — CHANGES_REQUESTED** [review]
Increment version and add a changelog.

File: SensateIoT.SmartEnergy.Dsmr.Parser.Common/Models/Telegram.cs (hunk 1 above)
Comment (michelmegens) [reply SensateIoT.SmartEnergy.Dsmr.Parser.Common/Models/Telegram.cs]: Fixed. Pulled the indentation type of this line check with the rest of the file.

**Review by sensateiot-cicd — APPROVED** [review]
Approved!

**Pull request merged**